### DISCUSSION TITLE
Highlighter pen

### DISCUSSION THREAD

**Discussion by nachiketkanore** [flxzt/rnote#828] [General]
Is there any option to highlight text in pdf for annotations similar to a brush/pen with custom opacity?

**Comment by flxzt** [comment]
Currently there is no free-draw pen for that (the graphics API that is currently used does not support different blend modes unfortunately). However, some users use this workaround: Use the line shape with a thick width and a half-transparent color to mark in a straight line.

**Reply by RexDoesntCode** [reply]
Thank you for working on Rnote.
Don't you think this workaround is very inefficient and annoying?
It would be better if we at least had a special pen that automatically draws with a transparent version of the color we choose from the palette (raising the alpha to an adjustable level).
Or, if this is very complicated or not possible, let the user add more colors to the palette so they can add transparent colors themselves.
